example/ios/ThermalReceiptPrinterExample/Info.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>UISupportedExternalAccessoryProtocols</key>
```

Comment:
Query:
What is about https://github.com/star-micronics/react-native-star-io10#4-apple-approval-process-for-star-mfi-applications?

It is said int he documentation, that for bluetooth this is needed

Reply:
We will need to submit this before releasing to the App Store